proto/federated_service/v1alpha1/federated_service.proto

```diff
@@ -13,7 +13,7 @@ message FederatedService {
   string namespace = 3;
   string cluster_name = 4;
   string trust_domain = 5;
-  string gateway_ip = 6;
+  map<string, string> gatewayIPs = 6;
```

Comment:
Unsure how much we should take from it, but the k8s API conventions recommends to avoid maps for collections and use a list of maps instead. I think the idea is that the key could then change over time if necessary.

We could also call the new field `gateways` to allow for additional info in future. And for backwards compat we could deprecate but keep the existing field for a while.

Reply:
That's a good thought.

How about this instead? 
https://github.com/cofide/cofide-api-sdk/pull/46/commits/cb0ed7e57004548d5e26351b450735c38b2139d1

That feels a bit cleaner, and it'd be useful to have it more structured for use in the agent.